src/main/java/com/netflix/imflibrary/st2067_2/CompositionModels/IMFCoreConstraintsChecker_st2067_2_2013.java

```diff
@@ -0,0 +1,172 @@
+package com.netflix.imflibrary.st2067_2.CompositionModels;
+
+import com.netflix.imflibrary.IMFErrorLogger;
+import com.netflix.imflibrary.exceptions.IMFException;
+import com.netflix.imflibrary.st2067_2.Composition;
+import com.netflix.imflibrary.utils.UUIDHelper;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.xml.bind.JAXBElement;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * A class that performs 2013 CoreConstraints st2067-2:2013 related checks on the elements of a Composition Playlist such as VirtualTracks, Segments, Sequences and Resources.
+ */
+public final class IMFCoreConstraintsChecker_st2067_2_2013 {
```

Comment:
It might be a good idea to add a private constructor to this class whose body is empty - to prevent instantiation